deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -131,6 +131,8 @@ spec:
               name: cni-bin-dir
             - mountPath: /host/etc/cni/net.d
               name: cni-net-dir
+            - mountPath: /host/etc/kubernetes/cni/net.d
```

Comment:
I checked the my k8s cluster: The k8s node (aka host) has not `/etc/kubernetes/cni/net.d`. 
It will break the gke